### PR TITLE
my homework 3

### PR DESCRIPTION
**Pull request by KawtherAl-Mutayliq**

### CODE REVIEW

File: HW3/src/main/kotlin/part2.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fun main (){
+sString()
+}
+fun sString(){
+   var phrase = "The quick brown fox jumps over the lazy dog"
```
Comment (MohamedFouad96) [inline HW3/src/main/kotlin/part2.kt:5]: i need only `brown fox jumps`  from this string

File: HW3/src/main/kotlin/part2.kt [hunk 2]
```diff
@@ -5,5 +5,5 @@ fun sString(){
    var phrase = "The quick brown fox jumps over the lazy dog"
 
     println(phrase.substring(0,15))
```
Comment (MohamedFouad96) [inline HW3/src/main/kotlin/part2.kt]: This line doesn't show the right text `brown fox jumps `

Comment (KawtherAl-Mutayliq) [reply HW3/src/main/kotlin/part2.kt]: It's work with me !!

Comment (KawtherAl-Mutayliq) [reply HW3/src/main/kotlin/part2.kt]: oh, ok sorry

**Comment by MohamedFouad96** [comment]
Thank you kawther, excellent work 👍🏻

**Comment by KawtherAl-Mutayliq** [comment]
Thank you

مرسل من Outlook Mobile<https://aka.ms/blhgte>
________________________________

**Pull request closed without merging**